src/Raven.Client/Documents/Operations/ETL/OLAP/OlapEtlConfiguration.cs

```diff
@@ -45,6 +45,32 @@ public override DynamicJsonValue ToJson()
 
             return json;
         }
+
+        internal bool Equals(OlapEtlConfiguration other)
+        {
+            if (other == null || RunFrequency != other.RunFrequency ||
+                CustomPartitionValue != other.CustomPartitionValue )
+                return false;
+
+            return EqualTables(other);
```

Comment:
Can we use https://github.com/ravendb/ravendb/blob/v5.1/src/Raven.Client/Extensions/EnumerableExtension.cs#L23 ?

Reply:
done